change hippocampal cells to relate to hippocampal formation

shawntanzk opened the pull request:
Fixes #1745
#gogoeditdiff

bvarner-ebi commented on src/ontology/cl-edit.owl:
```diff
@@ -19412,10 +19412,11 @@ SubClassOf(Annotation(oboInOwl:is_inferred "true") obo:CL_0002603 obo:CL_0000127
 
 # Class: obo:CL_0002604 (astrocyte of the hippocampus)
 
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "https://orcid.org/0000-0001-7258-9596"^^xsd:string) obo:IAO_0000115 obo:CL_0002604 "An astrocyte that is located in the hippocampus.")
```
As mentioned offline, the following appears in the unreasoned file:
<img width="445" alt="Screenshot 2022-11-24 at 14 10 41" src="https://user-images.githubusercontent.com/95643797/203804644-9b4579b2-8e5e-464c-927b-3e20ab1b2d8d.png">

shawntanzk replied:
Don't think its a problem, I think ORCIDS in this case are strings, not 100% sure, but anyway if its a problem down the road, its a simple SPARQL fix that should be applied to the whole ontology. QC should be up too if we need them as not strings

bvarner-ebi commented on src/ontology/cl-edit.owl:
```diff
@@ -19412,10 +19412,11 @@ SubClassOf(Annotation(oboInOwl:is_inferred "true") obo:CL_0002603 obo:CL_0000127
 AnnotationAssertion(oboInOwl:creation_date obo:CL_0002604 "2011-03-06T06:57:33Z"^^xsd:string)
 AnnotationAssertion(rdfs:label obo:CL_0002604 "astrocyte of the hippocampus"^^xsd:string)
-EquivalentClasses(obo:CL_0002604 ObjectIntersectionOf(obo:CL_0000127 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001954)))
+EquivalentClasses(obo:CL_0002604 ObjectIntersectionOf(obo:CL_0000127 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002421)))
```
[astrocyte of the hippocampus](http://purl.obolibrary.org/obo/CL_0002604) EquivalentTo [astrocyte](http://purl.obolibrary.org/obo/CL_0000127) and ([part_of](http://purl.obolibrary.org/obo/BFO_0000050) some [hippocampal formation](http://purl.obolibrary.org/obo/UBERON_0002421))

To consider: label is 'astrocyte of the hippocampus' and EA is astrocyte and part_of some 'hippocampal formation', so adding a synonym "astrocyte of the hippocampal formation" and/or comment as to why the label suggest something slightly different from the EA.

shawntanzk replied:
I like label as hippocampal astrocyte - this will probably be more consistent with literature too, esp with single cell work that is upcoming I think

bvarner-ebi commented on src/ontology/cl-edit.owl:
```diff
@@ -26138,7 +26138,7 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "CALOHA:TS-1257"^^xsd:string)
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "CALOHA:TS-1257"^^xsd:string) oboInOwl:hasRelatedSynonym obo:CL_1001580 "hippocampus neuroglial cell"^^xsd:string)
 AnnotationAssertion(rdfs:label obo:CL_1001580 "hippocampus glial cell"^^xsd:string)
-EquivalentClasses(obo:CL_1001580 ObjectIntersectionOf(obo:CL_0000125 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001954)))
+EquivalentClasses(obo:CL_1001580 ObjectIntersectionOf(obo:CL_0000125 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002421)))
```
Of note, it is quite surprising to not see 'hippocampus' as a label or exact synonym in Uberon, unless I am missing it....

bvarner-ebi reviewed (commented):
See inline comments.

shawntanzk commented:
sorry this took so long, slipped my radar and got pinged during my leave, but I think I've fixed all the issues :) Also not sure why hippocampal glial cell has related synonyms for things that should be exact, but have also fixed them while i was at it. Thanks

bvarner-ebi commented on src/ontology/cl-edit.owl:
```diff
@@ -27439,17 +27441,17 @@ SubClassOf(obo:CL_1001568 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_000531
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_1001569 "FMA:84777"^^xsd:string)
 AnnotationAssertion(rdfs:label obo:CL_1001569 "hippocampal interneuron"^^xsd:string)
-EquivalentClasses(obo:CL_1001569 ObjectIntersectionOf(obo:CL_0000099 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001954)))
+EquivalentClasses(obo:CL_1001569 ObjectIntersectionOf(obo:CL_0000099 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002421)))
```
@shawntanzk, I notice some EAs changed in this PR use the 'has soma location' object property while others, like this one, use 'part of'.
Is this intentional?

shawntanzk replied:
ah thanks for the spot, it should defs be 'has soma location' - will make the change. (guessing this is cause I changed my search and replace initially so missed it)

dosumis replied:
@shawntanzk to fix and merge ASAP

bvarner-ebi reviewed (approved):
Approving with the following caveats:

Text definitions are inconsistent in wording
(“located in” vs “part of” vs “of”) and not aligned with wording in equivalence axioms.

CL:0002604 ‘hippocampal astrocyte’
- missing dbxrefs on synonyms

CL:1001571 “hippocampus pyramidal neuron”
- synonyms may be “narrow” instead of “related”

CL:1001580 “hippocampal glial cell” 
- has synonym “hippocampus glial cells”, but not singular version like it used to have

shawntanzk commented:
> Text definitions are inconsistent in wording
(“located in” vs “part of” vs “of”) and not aligned with wording in equivalence axioms.

fixed

> CL:0002604 ‘hippocampal astrocyte’

> missing dbxrefs on synonyms

Not part of this ticket, should be addressed in another ticket if need be

> CL:1001571 “hippocampus pyramidal neuron”

> synonyms may be “narrow” instead of “related”

I see your point here, though arguments can be made that it is a related, but I think it doesn't over matter and narrow isn't wrong so I've changed it.

> CL:1001580 “hippocampal glial cells”

> has synonym “hippocampus glial cells”, but not singular version like it used to have

honestly super weird to need to have non singular when its cells vs cell - surely cell would suffice and any text mining things that needs both have bigger problems lol.  Anyway I've changed it all to singular.
